modules/core/02-client/keeper/events.go

```diff
@@ -0,0 +1,80 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/v3/modules/core/exported"
+)
+
+// EmitCreateClientEvent emits a create client event
+func EmitCreateClientEvent(ctx sdk.Context, clientID string, clientState exported.ClientState) {
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeCreateClient,
+			sdk.NewAttribute(types.AttributeKeyClientID, clientID),
+			sdk.NewAttribute(types.AttributeKeyClientType, clientState.ClientType()),
```

Comment:
missing consensus height attribute: `sdk.NewAttribute(clienttypes.AttributeKeyConsensusHeight, clientState.GetLatestHeight().String()),`

We should not be removing attributes in this pr. Relayers may be relying on this information